docs/guides/test-capability.md

```diff
@@ -0,0 +1,638 @@
+
+# Test Capability
+
+## Setup
+
+This guide assumes you have a project set up with Superface installed. If you need to set up a new project, please reference the [Setup Guide](./setup-the-environment.md).
+
+### Prerequisites
+
+- Existing profile
+- Existing [provider definition](./add-new-provider.md)
+- Existing [map between profile & the provider](./map-capability-to-provider.md)
+- [Superface CLI](/reference/cli) installed
+- [Superface OneSDK](/reference/one-sdk-js) installed
+
+## Configure existing profile and map {#configure-profile-map}
```

Comment:
So I think this section is redundant because if you have followed the guide, you have the profile and the provider setup. Maybe you should mention in the Prerequisites that the Profile and the Map must be local. This information could be perhaps useful in the troubleshooting section at the end of the page.

Reply:
I would also rather not document it here in this guide and let guide about super.json handle this.